Update teleport.yaml, copied from docs

eldios opened the pull request:
Update teleport.yaml, copied from docs version which includes newer settings

Commit:
Update teleport.yaml, copied from docs

Update teleport.yaml, copied from docs version which includes newer settings

gravitational-jenkins commented:
Can one of the admins verify this patch?

Commit:
Commented superfluous settings in a basic config

Commented superfluous settings in a basic config

benarent reviewed (changes requested):
Thanks for the PR @eldios ,  Should we also bump https://github.com/gravitational/teleport/blob/master/build.assets/pkg/init-scripts/systemd/system/teleport.service to match our current SystemD setup.

webvictim committed:
Merge branch 'master' into lele-update-teleport-yaml

webvictim commented:
ok to test

Commit:
Fixed stray option, causing syntax error

The kubernetes config section must be enabled or having just one line uncommented would cause issues

webvictim committed:
Merge branch 'master' into lele-update-teleport-yaml

benarent commented:
@eldios Could you please add in the updated SystemD module with this PR please.

Commit:
Removed stray new-line

Commit:
Changed to comply to online docs

benarent committed:
Merge branch 'master' into lele-update-teleport-yaml

Commit:
Revert "Changed to comply to online docs"

This reverts commit 3db4db7cf7d3689d9a09221e48fac75d9616f64f.

Commit:
Removed stray EOL space

Commit:
Updated build.asset systemd unit file to comply to docs

benarent committed:
Merge branch 'master' into lele-update-teleport-yaml
